### DISCUSSION TITLE
How to get real location ids in cluster

### DISCUSSION THREAD

**Discussion by mrosminin** [visgl/flowmap.gl#30] [Q&A] [answered]
In order to filter flows after click or on long hover as in example https://flowmap.blue/15kwLB4baXZ7jpip8q0JjgR6zDoS5Gt3gMLCTUAboQxk?v=46.719075%2C7.817990%2C7.51%2C0%2C0&a=0&as=1&b=1&bo=75&c=0&ca=1&d=1&fe=1&lt=1&lfm=ALL&col=Default&f=45

now onClick I get cluster ids in children '{[XXX]}' and real ids in children array.

**Comment by ilyabo** [comment, marked as answer]
I think this was answered here: https://github.com/visgl/flowmap.gl/discussions/17
